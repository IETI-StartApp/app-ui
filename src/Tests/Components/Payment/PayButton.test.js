import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';
import PayButton from '../../../components/Payment/index';

describe('Testing menu drawer', () => {

    const wrapper = shallow(<PayButton amount="200000" projectname="Proyecto tecnolÃ³gico" currency="cop" country="co" />)

    test('should load PayButton', () => {
        expect(wrapper).toMatchSnapshot();
    });

})
