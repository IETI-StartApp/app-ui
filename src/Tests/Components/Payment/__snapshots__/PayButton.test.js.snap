// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing menu drawer should load PayButton 1`] = `
<div
  id="PayButtonDiv"
/>
`;
